### PR TITLE
Adds some environmental sprites, resprites headstones

### PR DESCRIPTION
**Pull request by nefarious6th**
[sprites]

## About the PR 
Adds some environmental sprites (mostly rocks) in a new dmi under the icons/misc folder. (rocks.dmi)
Re-sprites headstones and adds some custom ones for various zones to have more uniqueness.

rocksandrelated

## Why's this needed? 
Perspective headstones and more azone flavor.
Wasn't sure if people would like the rocks really, so I didn't bother pasting the sprites I did over the boulder ones used in most places. New dmi also adds some geode sprites, stepping stones, crystals, and a few different rock varieties like sandstone that aren't used anywhere but are there if people want them. 
Don't know if anything in secret.dme would be affected by this; sorry in advance. Also sorry I can't spell lol

### CODE REVIEW

**Comment by notfrying1pans** [comment]
the old headstone sprites were stolen from mother 1 btw

File: maps/z2.dmm [hunk 1]
```diff
@@ -35473,6 +35474,7 @@
 "bIv" = (
 /obj/tombstone{
 	desc = "The inscription has eroded away.";
+	icon_state = "memorials";
```
Comment (Sovexe) [inline maps/z2.dmm:35477]: I'd rather see these get their own tombstone subtype instead of hard coding icon states onto the maps

**Review by Sovexe — CHANGES_REQUESTED** [review]

**Comment by ZeWaka** [comment]
Can you replace those old rocks with these rocks? They're stolen sprites and on the list.

File: maps/z2.dmm (hunk 1 above)
Comment (nefarious6th) [reply maps/z2.dmm:35477]: Sure; I tried to look for the tombstone code to do this but couldn't find it, but I will try again.

Comment (Sovexe) [reply maps/z2.dmm:35477]: it's in hallooween.dm I believe

Comment (nefarious6th) [reply maps/z2.dmm:35477]: Okay. I think I set it up correctly; I also just went through and made another headstone sprite and used that to replace the old one totally. Let me know if all that looks okay.

File: code/modules/holiday/halloween.dm [hunk 2]
```diff
@@ -51,6 +51,13 @@
 	var/robbed = 0
 	var/special = null //The path of whatever special loot is robbed from this grave.
 
+	memorial
+		name = "memorial marker"
+		//desc = "Here lies Tango N. Vectif, killed by a circus bear.  RIP." // changing for spawnability
```
Comment (ZeWaka) [inline code/modules/holiday/halloween.dm:56]:
```suggestion
```

**Comment by nefarious6th** [comment]
> Can you replace those old rocks with these rocks? They're stolen sprites and on the list.
Also I'm not ignoring this, I will get to this as soon as I can!

File: maps/z2.dmm [hunk 3]
```diff
@@ -41902,12 +41903,12 @@
 /area/afterlife/heaven)
 "bZV" = (
 /obj/tombstone{
+<<<<<<< HEAD
```
Comment (ZeWaka) [inline maps/z2.dmm:41906]: u fucked up on merging with all this stuff

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: maps/z2.dmm (hunk 3 above)
Comment (nefarious6th) [reply maps/z2.dmm:41906]: 🙀 will fix it when I re-run the linters later tonight

**Review by Sovexe — APPROVED** [review]

**Pull request merged**